spec/javascripts/components/notes/note_spec.js

```diff
@@ -1,59 +1,69 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render } from "@testing-library/react";
+import { screen } from '@testing-library/dom';
+import userEvent from "@testing-library/user-event";
 import NoteComponent from 'components/notes/Note';
 import Note from 'models/note.js';
 
-describe('<Note />', function() {
-  let note;
+window.document.getSelection = jest.fn()
 
-  beforeEach(function() {
-    sinon.stub(I18n, 't');
-    sinon.stub(window.md, 'makeHtml');
-    note = new Note({note: 'Test Note'});
-  });
+global.document.createRange = () => ({
+  setStart: () => { },
+  setEnd: () => { },
+  commonAncestorContainer: {
+    nodeName: 'BODY',
+    ownerDocument: document,
+  },
+});
+
+describe('<Note />', () => {
+  const setup = (disabled) => {
+    let note;
+
+    jest.spyOn(I18n, 't');
+    jest.spyOn(window.md, 'makeHtml');
```

Comment:
We should avoid doing these two in `setup` instead of `beforeEach` because if we don't call `setup` in some of the tests the `afterEach` will try to clean something that is not a mock.